res/values-ku/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="Players">"lîstikvan"</string>
+    <string name="Players">"lîstikvanên"</string>
```

Comment:
Please correct the first letter "L" and turn it into capital letter, then it will be elligible

Reply:
Done